app/services/idv/steps/back_image_step.rb

```diff
@@ -0,0 +1,77 @@
+module Idv
+  module Steps
+    class BackImageStep < DocAuthBaseStep
+      def call
+        good, data = assure_id.post_back_image(image.read)
+        return failure(data) unless good
+
+        failure_data, data = verify_back_image
+        return failure_data if failure_data
+
+        extract_pii_from_doc_and_perform_resolution(data)
+      end
+
+      private
+
+      def form_submit
+        Idv::ImageUploadForm.new(current_user).submit(permit(:image))
+      end
+
+      def extract_pii_from_doc_and_perform_resolution(data)
+        pii_from_doc = Idv::Utils::PiiFromDoc.new(data).
+                       call(flow_session[:ssn], current_user.phone)
+        result = perform_resolution(pii_from_doc)
+        if result.success?
+          step_successful(pii_from_doc)
+        else
+          flow_session[:matcher_pii_from_doc] = pii_from_doc
+        end
+      end
+
+      def step_successful(pii_from_doc)
+        mark_step_complete(:doc_failed) # skip doc failed
+        save_legacy_state(pii_from_doc)
+      end
+
+      def save_legacy_state(pii_from_doc)
+        skip_legacy_steps
+        idv_session['params'] = pii_from_doc
+        idv_session['applicant'] = pii_from_doc
+        idv_session['applicant']['uuid'] = current_user.uuid
+      end
+
+      def skip_legacy_steps
+        idv_session['profile_confirmation'] = true
+        idv_session['vendor_phone_confirmation'] = true
+        idv_session['user_phone_confirmation'] = true
+        idv_session['address_verification_mechanism'] = 'phone'
+        idv_session['resolution_successful'] = 'phone'
+      end
+
+      def perform_resolution(pii_from_doc)
+        result_id = SecureRandom.uuid
+        Idv::ProoferJob.perform_now(
```

Comment:
This will need to be changed since we're getting rid of the background jobs. Additionally, we'll want to check w/ AAMVA here if the jurisdiction is supported.